### ISSUE TITLE
nvda support

### ISSUE THREAD

**Issue by amirh123123** [libretro/RetroArch#14396]
hi i want use my screan reader to read menu i use windows text to speach  but i have nvda and i want work with nvda some menus work with nvda and some menus work with win text to speach i want all menus work with nvda or text to speach thanks

**Comment by BarryJRowe** [comment]
@amirh123123 Can you give a more detailed description of what menus work with nvda, and what only works with text to speech?

**Comment by amirh123123** [comment]
in main menu and quick menu f 1 work with text to speach but in file or desktup menu work with nvda

**Comment by BarryJRowe** [comment]
Thanks, we're working on it.

**Comment by gouchi** [comment]
It seems we were shipping on Windows `NVDAcontrolerclient.dll` but now it seems not case any more ?

And we have [HAVE_NDA](https://github.com/libretro/RetroArch/blob/master/qb/config.params.sh#L5) enabled.

Source : [1](https://discord.com/channels/184109094070779904/723004014140325949/1033156634886819870) [2](https://blindhelp.net/software/retroarch-latest)

**Comment by gouchi** [comment]
It seems was present in version 1.9.0 but from version 1.9.1 is missing. 

[Source](https://discord.com/channels/184109094070779904/723004014140325949/1033691691170930718)

About `nvdaControllerClient64.dll` related [commit](https://github.com/libretro/RetroArch/commit/e7082d3e5b69490a055469c346a5c3963b0f825d).

**Comment by k-kolev1985** [comment]
Hello,

I’m a visually impaired person and therefore use RetroArch wit its accessibility mode enabled and in combination with the NVDA screen reader.

Recently I’ve updated my portable copy of RetroArch to the latest stable version (I think it is 1.13.0).

Since then, the speech output to NVDA stopped working. Now, RetroArch outputs speech to SAPI5 instead.

With a previous version (a build from the 17th of October), it was working as it should.

Any ideas what’s going on? Am I suffering from the same issue - missing "nvdaControllerClient64.dll" file?

Test environment:
• Operating system: Windows 11 Pro version 22H2 (build 22621.674), 64-bit, in Bulgarian with all locale settings set to “Bulgarian”.
• RetroArch version: 1.13.0 (a build from November, if I’m not mistaking).
• NVDA version: 2022.3.2.
• Processor: Intel Core i5-9400F at 2.90GHz.
• RAM Memory: 16.00GB.
• Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
• Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

Thanks in advance!

**Comment by amirh123123** [comment]
hi i use scan directory but folders cant read with nvda

**Comment by amirh123123** [comment]
i scan directory but dont read with accessibility text to speach

**Comment by gouchi** [comment]
Hi @k-kolev1985 

I checked RetroArch [version 1.14.0](https://buildbot.libretro.com/stable/1.14.0/windows/x86_64/) and there is the dll

``` 
7z l ./RetroArch.7z | grep nvdaControllerClient64.dll
2022-12-13 23:50:58 ....A       153600               RetroArch-Win64/nvdaControllerClient64.dll
```

**Comment by gouchi** [comment]
@amirh123123 This issue should be closed as it has been fixed.

@k-kolev1985 It should be working now.

Thank you.

**Comment by k-kolev1985** [comment]
@gouchi Yes, I can confirm that with the latest RetroArch it is working as expected.
